<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>381</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_Title">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bracket Balance Program UI </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_TextEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>261</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Type in string with brackets and run:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Type in a string with brackets and click Run...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Result">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>64</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Result:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ResultArea">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Run">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Clean">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>110</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Clean</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui> 
